src/layout/layout.module.css

```diff
@@ -57,6 +68,113 @@
 .header__nav a:hover {
   color: rgba(0, 0, 0, 0);
 }
+
+.container {
```

Comment:
Think this can be named something more specific. Like `buttonContainer`

Reply:
agreed!

Reply:
Can be even more specific IMO.